lib/mix/tasks/nerves/new.ex

```diff
@@ -387,4 +400,7 @@ defmodule Mix.Tasks.Nerves.New do
   defp generate_source_date_epoch() do
     DateTime.utc_now() |> DateTime.to_unix()
   end
+
+  defp get_mappings(true), do: @new_common ++ @new_umbrella
+  defp get_mappings(false), do: @new_common ++ @new
```

Comment:
```suggestion
  defp get_mappings(_false), do: @new_common ++ @new
```

I get an error locally because when outside of an umbrella, this is actually `nil` (due to just how the check is setup)

```
tmp git:(TehSnappy/main) ✗ mix nerves.new oldy
** (FunctionClauseError) no function clause matching in Mix.Tasks.Nerves.New.get_mappings/1    
    
    The following arguments were given to Mix.Tasks.Nerves.New.get_mappings/1:
    
        # 1
        nil
    
    Attempted function clauses (showing 2 out of 2):
    
        defp get_mappings(true)
        defp get_mappings(false)
    
    lib/mix/tasks/nerves/new.ex:405: Mix.Tasks.Nerves.New.get_mappings/1
    lib/mix/tasks/nerves/new.ex:226: Mix.Tasks.Nerves.New.run/4
    (mix 1.14.2) lib/mix/task.ex:421: anonymous fn/3 in Mix.Task.run_task/4
    (mix 1.14.2) lib/mix/cli.ex:84: Mix.CLI.run_task/2
```